Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -189,16 +189,16 @@ ErrorOr<void> PropertiesWindow::create_general_tab(GUI::TabWidget& tab_widget, b
             : human_readable_size_long(st.st_size, UseThousandsSeparator::Yes));
 
     auto* owner = general_tab.find_descendant_of_type_named<GUI::Label>("owner");
-    owner->set_text(String::formatted("{} ({})", owner_name, st.st_uid).release_value_but_fixme_should_propagate_errors());
+    owner->set_text(TRY(String::formatted("{} ({})", owner_name, st.st_uid)));
```

Comment:
Most of these (at least the string formatting ones, I'm not sure about loading images/creating widgets) seem like they should be `MUST`? cf. #20405

Reply:
Yeah, on second thought. `TRY` isn't the right thing to do here, but `MUST` will cause a crash if we are converting a ByteString that isn't valid UTF-8.

It seems like the right thing to do would be to have some kind of fallback mechanism that would replace invalid UTF-8 characters with a question mark or box or something along those lines.

For now I've reverted the changes where unsanitized ByteStrings were being converted to String with `TRY`. I changed the instances that we can be sure are safe to use `MUST` instead.

I can revisit it when I come up with something better.